packages/components/radio/src/use-radio.ts

```diff
@@ -58,6 +58,12 @@ export const useRadio = (
     computed(() => isGroup.value && isPropAbsent(props.value))
   )
 
+  function cleared() {
```

Comment:
```suggestion
  function handleClear() {
```
I think `handleClear` is better.

Reply:
It makes sense